knowledge-base/button-change-icons-position.md

```diff
@@ -23,22 +23,54 @@ res_type: kb
 
 ## Description
 
-How can I show an icon in a Telerik UI for Blazor Button on the right side of the text instead of on the left side?
-How can I also include more than one icon in the Button?
+This KB article answers the following questions:
+
+* How can I show an icon in a Telerik UI for Blazor Button on the right side of the text instead of on the left side?
+* How can I show an icon in a Telerik UI Blazor Button on the top and bottom side of the text?
+* How can I also include more than one icon in the Button?
 
 ## Solution
 
-You can change the position of the [icons]({%slug common-features-icons%}) and add more icons in the button by nesting the `TelerikIcon` in the `TelerikButton` content.
+You can change the position of the [icons]({%slug common-features-icons%}) and add more icons in the buttons by the following approaches:
+
+* Nesting the `TelerikIcon` in the `TelerikButton` content.
+* Use CSS to position an icon either above or below the text content.
 
 >caption Reposition the Button icon and add more than one icon to the component
 
 ![Telerik UI for Blazor Button with two icons](images/button-change-icon-position-example.png)
+![Telerik UI for Blazor Button with  icon on top](images/button-change-icon-position-top-example.png)
+![Telerik UI for Blazor Button with  icon on bottom](images/button-change-icon-position-bottom-example.png)
```

Comment:
We try to remove screenshots, rather than add new ones. Normally, we only leave screenshots that are animated and show the component behavior as a process.

Reply:
It seems that there are two spaces between "with" and "icon".